src/packages/quickenter/doc.zh-TW.md

```
@@ -0,0 +1,140 @@
+| 參數 | 説明 | 類型 | 默認值 |
+| --- | --- | --- | --- |
+| title | 選項標題 | `string` | `-` |
+| icon | 選項圖標 | `ReactNode` | `-` |
+| type | 選項類型 | `string` | `-` |
+| url | 跳轉鏈接 | `string` | `-` |
+
+## 主題定製
+
+### 樣式變量
+
+組件提供了下列 CSS 變量，可用於自定義樣式，詳情請參考 [ConfigProvider 組件](#/zh-CN/component/configprovider)。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)